src/viking.js

```diff
@@ -1,14 +1,83 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength
+  }
+  receiveDamage(damage) {
+this.health = this.health - damage
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor (name, health, strength) {
+    super(health, strength);
+    this.name = name
+  } 
+  receiveDamage(damage) {
+this.health= this.health - damage
+  if (this.health > 0 ) {
+    return `${this.name} has received ${damage} points of damage`
+  } else if (this.health === 0){ 
+return `${this.name} has died in act of combat`
+  }
+  }
+  battleCry() {
+    return "Odin Owns You All!";
+  }
+
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier { 
+  receiveDamage(damage) { 
+    this.health= this.health - damage
+    if (this.health > 0 ) {
+      return `A Saxon has received ${damage} points of damage`
+    } else if (this.health === 0) {
+      return `A Saxon has died in combat`
+    }
+  }
+}
 
 // War
-class War {}
+class War {
+  constructor(){
+    this.vikingArmy = []
+    this.saxonArmy = []
+  }
+  addViking(Viking){
+    this.vikingArmy.push(Viking)
+  }
+  addSaxon(Saxon){
+    this.saxonArmy.push(Saxon)
+  }
```

Comment:
i recommend sticking to convention and starting each variable name with lower case ,
`viking` , `saxon` etc